rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml

```
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to exploit privilege escalation vulnerabilities related to the Adobe Acrobat Reader
+PrivilegedHelperTool responsible for installing updates. For more information, refer to CVE-2020-9615, CVE-2020-9614 and
+CVE-2020-9613 and verify that the impacted system is patched.
+"""
+false_positives = ["Trusted system or Adobe Acrobat Related processes."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Child Process of Adobe Acrobat Reader Update Service"
+references = [
+    "https://rekken.github.io/2020/05/14/Security-Flaws-in-Adobe-Acrobat-Reader-Allow-Malicious-Program-to-Gain-Root-on-macOS-Silently/",
+]
+risk_score = 73
+rule_id = "f85ce03f-d8a8-4c83-acdc-5c8cd0592be7"
+severity = "high"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Privilege Escalation"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+  process.parent.name:com.adobe.ARMDC.SMJobBlessHelper and
+  user.name:root and
+  not process.executable:
+  (
+    /Library/PrivilegedHelperTools/com.adobe.ARMDC.SMJobBlessHelper or
+    /usr/bin/codesign or
+    /private/var/folders/zz/*/T/download/ARMDCHammer or
+    /usr/sbin/pkgutil or
+    /usr/bin/shasum or
+    /usr/bin/perl* or
+    /usr/sbin/spctl or
+    /usr/sbin/installer
+  )
```

kql query whitespace is normalized at package generation time and so it is better to split them in a specific manner

```suggestion
event.category:process and event.type:(start or process_started) and
  process.parent.name:com.adobe.ARMDC.SMJobBlessHelper and
  user.name:root and
  not process.executable: (/Library/PrivilegedHelperTools/com.adobe.ARMDC.SMJobBlessHelper or
                           /usr/bin/codesign or
                           /private/var/folders/zz/*/T/download/ARMDCHammer or
                           /usr/sbin/pkgutil or
                           /usr/bin/shasum or
                           /usr/bin/perl* or
                           /usr/sbin/spctl or
                           /usr/sbin/installer)
```